bokehjs/src/lib/models/glyphs/webgl/types.ts

```diff
@@ -0,0 +1,150 @@
+import {AttributeConfig, Texture2D, Vec2, Vec4} from "regl"
+
+
+interface CommonProps {
```

Comment:
For the sake of consistency with the rest of bokehjs, we prefer `type` over `interface` in general, and use `interface` only were truly necessary.

Reply:
OK, I've switched from
```typescript
interface A extends B, C {... blah blah ...}
```
to
```typescript
type A = B & C & {... blah blah ...}
```